ci/pipeline.yml

```diff
@@ -97,11 +97,11 @@ jobs:
   plan:
   - in_parallel:
     - get: broker-src
-      passed: [acceptance-tests-staging]
+      passed: [deploy-staging]
```

Comment:
Same question as other PR - are these being renamed to account for the new broker?

Reply:
This parameter says "`broker-src` must pass the referenced job before being valid for this job"
since we removed `acceptance-tests-staging`, referencing it here creates an invalid config, so we have to change it to reference the staging deployment job.

Reply:
Got it, thank you!